kANe/v5p1/1_Vratas_and_Utsavas/02_Vrata_in_Vedas_sUtras_and_smRtis.md

```diff
@@ -244,154 +244,151 @@ It is no doubt true that in most vratas brāhmaṇas are to be fed, but one shou
 
 There were certain differences between Vedic sacrifices and vratas. The promised reward of most sacrifices was heaven, while Purāṇas promised that most vratas would confer tangible benefits in this very life. Further, vratas could be performed by: all, including sūdras, maidens, married women, widows, even prostitutes.
 
-There were vratas for the brahmaçõrin (called Veda-vratas ) and for the spātaka. These have been already dealt with in H. of Dh. vol. II, pp. 370-374 and pp. 412-415 respectively. That
+There were vratas for the _brahmaçõrin_ (called Veda-vratas ) and for the sṇātaka. These have been already dealt with in H. of Dh. vol. II, pp. 370-374 and pp. 412-415 respectively.
 
-112. देवत्वं देवताः प्राक्षा नियमानियमान्विते। तारारूपा ज्वलन्त्येते नियमानुसयोधने। me q. by . (op ) I. p. 320, a. 4. folio 9 a.
+[^112]: देवत्वं देवताः प्राप्ता नियमान्नियमान्विते। तारारूपा ज्वलन्त्येते नियमानुसयोधने। स्कन्द q. by हे. (on व्रत ) I. p. 320, व्र. प्र. folio 9 a.
 
 46
 
 
 
 
 
-vratas (in the medieval sense ) existed even in the first centuries before and after the Christian era follows from the Ap. Dh. S. and from the dramas of Kālidāsa, from the Mrcchakatika and the Ratnāyali. The Ap. Dh. S. II. 8. 18 prescribes a vrata for one year which is equal. in its merit to brahmacārivrata for 48 years and Ap. Dh. S. II. 8. 18-19 to Ap. Dh. S. II. 8. 20 prescribe two modes of a vrata to be begun on Tisya (Pusya) naksatra to be observed by a person desirous of prosperity. Vide S. B. E. vol. II, pp. 145– 148. Hemādri (vrata II. .p. 628 quotes Āp. Dh. S. II. 8. 20.3-9), explains the sutras and calls this Āpastambokta Pusyavrata.
+That vratas (in the medieval sense ) existed even in the first centuries before and after the Christian era follows from the Āp. Dh. S. and from the dramas of Kālidāsa, from the Mṛcchakaṛika and the Ratnāyali. The Āp. Dh. S. II. 8. 18 prescribes a vrata for one year which is equal in its merit to brahmacārivrata for 48 years and Ap. Dh. S. II. 8. 18-19 to Āp. Dh. S. II. 8. 20 prescribe two modes of a vrata to be begun on Tisya (Puśya) naksatra to be observed by a person desirous of prosperity. Vide S. B. E. vol. II, pp. 145– 148. Hemādri (vrata II. .p. 628 quotes Āp. Dh. S. II. 8. 20.3-9), explains the sutras and calls this Āpastambokta Pusyavrata.
```

Comment:
Mṛcchakaṭika

Reply:
That vratas (in the medieval sense ) existed even in the first centuries before and after the Christian era follows from the Āp. Dh. S. and from the dramas of Kālidāsa, from the Mṛcchakaṭika and the Ratnāyali. The Āp. Dh. S. II. 8. 18 prescribes a vrata for one year which is equal in its merit to brahmacārivrata for 48 years and Ap. Dh. S. II. 8. 18-19 to Āp. Dh. S. II. 8. 20 prescribe two modes of a vrata to be begun on Tisya (Puśya) naksatra to be observed by a person desirous of prosperity. Vide S. B. E. vol. II, pp. 145– 148. Hemādri (vrata II. .p. 628 quotes Āp. Dh. S. II. 8. 20.3-9), explains the sutras and calls this Āpastambokta Pusyavrata.
Corrected.

Reply:
Please correct in the file and push.